native-new/src/layout/yoga.rs

```diff
@@ -31,42 +33,42 @@ impl <'svc> YogaLayoutService {
     }
 
     pub fn append_child(&mut self, parent: Id, child: Id) {
-        let (parent, child) = self.yoga_nodes.get_two_muts(parent, child);
+        let (parent, child) = get_two_muts(&mut self.yoga_nodes, parent, child);
 
         let index = parent.get_child_count();
         parent.insert_child(child, index);
     }
 
     pub fn remove_child(&mut self, parent: Id, child: Id) {
-        let (parent, child) = self.yoga_nodes.get_two_muts(parent, child);
+        let (parent, child) = get_two_muts(&mut self.yoga_nodes, parent, child);
 
         parent.remove_child(child);
     }
 
     // easier with index rather than with Id
     pub fn insert_at(&mut self, parent: Id, child: Id, index: u32) {
-        let (parent, child) = self.yoga_nodes.get_two_muts(parent, child);
+        let (parent, child) = get_two_muts(&mut self.yoga_nodes, parent, child);
 
         parent.insert_child(child, index);
     }
 
     pub fn set_size(&mut self, id: Id, size: Size) {
-        self.yoga_nodes.get_mut(id).apply_styles(&vec![
+        self.yoga_nodes[id].apply_styles(&vec![
             FlexStyle::Width(size.0.into()),
             FlexStyle::Height(size.1.into()),
         ])
     }
 
     pub fn set_flex(&mut self, id: Id, flex: Flex) {
-        self.yoga_nodes.get_mut(id).apply_styles(&vec![
+        self.yoga_nodes[id].apply_styles(&vec![
```

Comment:
I think this is an alllocation. Maybe use tuple? or something that can be allocates on the stack?
Or im missing something?

Reply:
yep, it was possible to replace `&vec![...]` with simple `&[...]`
thx!